src/configs/YITHPlugins.config.js

```diff
@@ -43,112 +43,155 @@ export const YITHPluginsDefinitions = (props) => ({
         },
       ],
     },
     {
       Card: FeatureCard,
       shouldRender: () => true,
-      name: 'yith-woocommerce-ajax-search',
+      name: 'nfd_slug_yith_woocommerce_wishlist',
       assets: () => ({
-        Image: Search,
+        Image: WishList,
+        learnMoreUrl: YITH_WOOCOMMERCE_WISHLIST,
       }),
       text: ({ isActive }) => ({
-        title: __('YITH WooCommerce AJAX Search', 'wp-module-ecommerce'),
+        title: __('Wishlists', 'wp-module-ecommerce'),
         description: __(
-          'YITH WooCommerce AJAX Search is an effective and predictive real-time search engine on your store so users can quickly find products they want. Speed up your search and boost your sales.',
+          'Let customers add products to lists and share them with family and friends.',
           'wp-module-ecommerce'
         ),
-        actionName: isActive ? __('Manage', 'wp-module-ecommerce') : __('Enable', 'wp-module-ecommerce'),
-        slug: 'yith_wcas_panel',
+        actionName: isActive ? __('View Wishlists', 'wp-module-ecommerce') : __('Setup Wishlists', 'wp-module-ecommerce'),
+        slug: 'yith_wcwl_panel',
       }),
       state: defineFeatureState(),
       actions: {
         installFeature: createPluginInstallAction(
-          'yith-woocommerce-ajax-search',
-          11,
+          'nfd_slug_yith_woocommerce_wishlist',
+          12,
           props
         ),
       },
       queries: [
         {
           key: 'plugins',
-          selector: wcPluginStatusParser('yith-woocommerce-ajax-search'),
+          selector: wcPluginStatusParser('nfd_slug_yith_woocommerce_wishlist'),
         },
         {
           key: 'upsellOptions',
-          selector: findUpsellWithName('YITH WooCommerce AJAX Search'),
+          selector: findUpsellWithName('YITH WooCommerce Wishlist'),
         },
       ],
     },
     {
       Card: FeatureCard,
       shouldRender: () => true,
-      name: 'nfd_slug_yith_woocommerce_wishlist',
+      name: 'nfd_slug_wonder_cart',
       assets: () => ({
-        Image: WishList,
-        learnMoreUrl: YITH_WOOCOMMERCE_WISHLIST,
+        Image: WonderCart,
       }),
       text: ({ isActive }) => ({
-        title: __('YITH WooCommerce Wishlist', 'wp-module-ecommerce'),
+        title: __(
+          'WonderCart',
+          'wp-module-ecommerce'
+        ),
+        primaryUrl: "admin.php?page=bluehost#/store/sales_discounts",
         description: __(
-          'Allow your customers to create lists of products they want and share them with family and friends.',
+          'Create custom upsell, cross-sell and other promotional campaigns to generate more sales.',
           'wp-module-ecommerce'
         ),
-        actionName: isActive ? __('Manage', 'wp-module-ecommerce') : __('Enable', 'wp-module-ecommerce'),
-        slug: 'yith_wcwl_panel',
+        actionName: isActive ? __('Create a Campaign', 'wp-module-ecommerce') : __('Create a Campaign', 'wp-module-ecommerce'),
```

Comment:
do we need a condition here, as it is always same

Reply:
done